src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/Int32/Int32FrozenDictionary.AlternateLookup.cs

```diff
@@ -10,8 +10,29 @@ namespace System.Collections.Frozen
 {
     internal sealed partial class Int32FrozenDictionary<TValue>
     {
-        /// <inheritdoc/>
-        private protected override ref readonly TValue GetValueRefOrNullRefCore<TAlternateKey>(TAlternateKey key)
+
+        private protected override AlternateLookupDelegate<TAlternateKey> GetAlternateLookupDelegate<TAlternateKey>()
```

Comment:
Can you put back the `/// <inheritdoc/>`s and make sure the base is appropriately documented?